src/com/facebook/buck/android/SmartDexingStep.java

```diff
@@ -205,10 +206,17 @@ private void runDxCommands(ExecutionContext context, Multimap<Path, Path> output
     List<Step> dxSteps = generateDxCommands(filesystem, outputToInputs);
 
     List<Callable<Void>> callables = Lists.transform(dxSteps,
-        step -> () -> {
-          stepRunner.runStepForBuildTarget(context, step, Optional.empty());
-          return null;
-        });
+        new Function<Step, Callable<Void>>() {
+	  @Override
+          public Callable<Void> apply(Step step) {
+	    try {
+              stepRunner.runStepForBuildTarget(context, step, Optional.empty());
+            } catch (Exception exc) {
+	      // Ignore
+            }
+            return null;
+          }
+	});
```

Comment:
The base version doesn't compile for me, after adding the EP 2.0.17 on the classpath,
even without using the EP itself.